src/en/comicextra/src/eu/kanade/tachiyomi/extension/en/comicextra/ComicExtra.kt

```diff
@@ -23,75 +23,78 @@ class ComicExtra : ParsedHttpSource() {
+    override fun popularMangaSelector() = "div.eg-box"
 
-    override fun latestUpdatesSelector() = "div.hl-box"
+    override fun latestUpdatesSelector() = "ul.line-list"
 
-    override fun popularMangaRequest(page: Int) = GET("$baseUrl/popular-comic/$page", headers)
+    override fun popularMangaRequest(page: Int) = GET("$baseUrl/popular-comics?page=$page", headers)
 
-    override fun latestUpdatesRequest(page: Int) = GET("$baseUrl/comic-updates/$page", headers)
+    override fun latestUpdatesRequest(page: Int) = GET("$baseUrl/comic-updates?page=$page", headers)
 
     override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
-        return if (query.isNotBlank()) {
-            val url = "$baseUrl/search".toHttpUrl().newBuilder().apply {
-                addQueryParameter("keyword", query)
-                if (page > 1) addQueryParameter("page", page.toString())
-            }.build()
-            GET(url, headers)
-        } else {
-            var url = baseUrl
+        val url = "$baseUrl/advanced-search".toHttpUrl().newBuilder().apply {
+            if (query.isNotBlank()) addQueryParameter("key", query)
+            if (page > 1) addQueryParameter("page", page.toString())
             filters.forEach { filter ->
                 when (filter) {
-                    is GenreFilter -> url += "/${filter.toUriPart()}"
+                    is GenreGroupFilter -> {
+                        with(filter) {
+                            addQueryParameter("wg", included.joinToString("%20"))
+                            addQueryParameter("wog", excluded.joinToString("%20"))
+                        }
+                    }
+                    is StatusFilter -> addQueryParameter("status", filter.selected)
                     else -> {}
                 }
             }
-            GET(url + if (page > 1) "/$page" else "", headers)
-        }
+        }.build()
+        return GET(url, headers)
     }
 
     override fun popularMangaFromElement(element: Element) = SManga.create().apply {
-        setUrlWithoutDomain(element.select("div.mb-right > h3 > a").attr("href"))
-        title = element.select("div.mb-right > h3 > a").text()
+        setUrlWithoutDomain(element.select("a.eg-image").attr("href"))
+        title = element.select("div.egb-right a").text()
         thumbnail_url = element.select("img").attr("src")
     }
 
     override fun latestUpdatesFromElement(element: Element) = SManga.create().apply {
-        setUrlWithoutDomain(element.select("div.hlb-t > a").attr("href"))
-        title = element.select("div.hlb-t > a").text()
-        thumbnail_url = fetchThumbnailURL(element.select("div.hlb-t > a").attr("href"))
+        setUrlWithoutDomain(element.select("a.big-link").attr("href"))
+        title = element.select(".big-link").first()?.text().toString()
+        thumbnail_url = fetchThumbnailURL(element.select("a.big-link").attr("href"))
```

Comment:
remove this, as making n calls for n entries isn't good. latest updates can remain without thumbnail